packages/sandcastle/src/App.tsx

```diff
@@ -678,13 +609,36 @@ Sandcastle.addToolbarMenu(${variableName});`,
     const html = htmlEditorRef.current.getValue();
     console.log([code, html]);
 
-    const base64String = makeCompressedBase64String([code, html]);
+    const base64String = makeCompressedBase64String({ code, html });
 
     // const shareUrl = `${getBaseUrl()}#c=${base64String}`;
     const shareUrl = `#c=${base64String}`;
     window.history.replaceState({}, "", shareUrl);
   }
 
+  function openStandalone() {
+    if (!jsEditorRef.current || !htmlEditorRef.current) {
+      // can't find next highest if there's no code yet
+      return;
+    }
+    const code = jsEditorRef.current.getValue();
+    const html = htmlEditorRef.current.getValue();
+
+    let baseHref = getBaseUrl();
+    const pos = baseHref.lastIndexOf("/");
+    baseHref = `${baseHref.substring(0, pos)}/gallery/`;
+
+    console.log([code, html]);
```

Comment:
Here as well.

Reply:
I agree long term these need to be removed and may seem excessive but I've been using these logs to debug and keep track of how the app is loading things. They've been especially helpful as I shift or try different load processes with monaco and the gallery. I will do a cleanup pass at some point but think they're fine to stay for now.